tabular/tests/unittests/resource_allocation/test_total_resource_allocation.py

```diff
@@ -137,6 +144,19 @@ def test_bagged_model_without_total_resources_but_with_ensemble_resources(mock_s
         k_fold = 2
         resources = bagged_model._preprocess_fit_resources(k_fold=k_fold)
         resources.pop('k_fold')
+        assert resources == {'num_cpus': 1, 'num_gpus': 0}
+        
+        
+def test_bagged_model_without_total_resources_and_without_model_resources(mock_system_resources_ctx_mgr, mock_num_cpus, mock_num_gpus):
+    with mock_system_resources_ctx_mgr(num_cpus=mock_num_cpus, num_gpus=mock_num_gpus):
+        model_base = DummyModel()
+        bagged_model = DummyBaggedModel(
+            model_base,
+            hyperparameters={}
+        )
+        k_fold = 2
+        resources = bagged_model._preprocess_fit_resources(k_fold=k_fold)
+        resources.pop('k_fold')
```

Comment:
Why is this hard-coded? Wouldn't it be way easier to just have this be in the arguments of the test?